main/config/logback.xml

```diff
@@ -31,6 +31,7 @@
       <compressLoggerName>true</compressLoggerName>
       <jacksonModule class="com.fasterxml.jackson.datatype.guava.GuavaModule" />
       <jacksonModule class="com.fasterxml.jackson.datatype.jdk8.Jdk8Module" />
+      <jacksonModule class="com.fasterxml.jackson.datatype.jsr310.JavaTimeModule" />
```

Comment:
It should go in `main/config/logback-console.xml` as well.

Reply:
☑️